.github/workflows/docker.yml

```diff
@@ -11,6 +11,12 @@ on:
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
       - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
+  workflow_dispatch:
```

Comment:
stills needs to check this.

Reply:
we use it for new proto docker images, you can copy that

Reply:
Yes, but still need to add it somewhere here, while keeping the behavior.

```yml
          tags: |
            type=raw,value=latest,enable={{is_default_branch}}
            type=semver,pattern={{version}}
            type=semver,pattern=latest-{{major}}.{{minor}}
          flavor: |
            latest=false
```